### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Auth Exp My VA: Payment info section v2

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#43330]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
As this goes to code, make sure the "Recent unpaid debts" and "Recent payments deposited" are both coded as H3's, even if the CSS styling is different from the default styling. (This looks right in the Sketch file, but I just want to make sure it doesn't get lost!)

**Should:**
There was lots of discussion in the meeting today about how the lists of debts/payments should be structured, and in the conversation we didn't land on a precise recommendation. I'll continue to dodge on that because it's actually a very difficult question with no clear answer!

Instead, I'll make this "should" recommendation: Explore multiple ways of structuring the content before the design is finalized for coding, and work with the PTEMS accessibility folks (@davidakennedy, @AngelaFowler82, and @TiffanyPender) on building lightweight HTML prototypes for testing. And please feel free to include me as well!

The accessibility goals for the debt/payments are:
- Debt/payment lists that are easy to navigate through for screen reader users. Possible implementations would be coded using H4 headings for each item, coded as an unordered list, or coded as a data table. The H4 headings and the unordered list routes may look very similar to what's currently in your design, and the table would probably be a bigger departure.
- All of the related information is clearly grouped together, both visually and for screen reader users.
- The most important information is presented first, especially as announced by screen readers; or, in the case of a data table, it's easy for a screen reader user to navigate the table to quickly find the most important information. 

For that last bullet point: What qualifies as "most important" is probably something that will be revealed through usability testing. It might still be worth adding something to the conversation guide to try to determine what Veterans are keying in on when looking at the list. When scanning through the list, do they care most about what the debt/payment is for, what the due date is, or what the dollar amount is?

In the meeting today I suggested prioritizing the dollar amount ahead of what the debt/payment is for --- especially seeing that extra-long copay example. My gut feeling is that's probably what most vets are going to be looking for first and that should be the first thing announced by screen readers, but that's just a guess on my end. I'd definitely recommend watching for that in testing, and doing testing with screen reader users if possible.

**Consider:**
- Consider adding a +/- or some other indicator to distinguish debts and payments. This isn't necessary based on how the information is structured, but it may be useful to make them more explicit, especially for colorblind users. This is probably something you can wait on until after usability testing. If users aren't confused (especially colorblind users) then it's not something I would lose sleep over.
- Out of scope but something for future iterations: Consider standardizing heading styles on My VA, either sticking to default styling or having a consistent custom styling across all the sections. The coded heading level matters most for screen reader users, but sighted non-screen reader users also benefit from seeing a clear visual hierarchy.

## Platform directions
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by davidakennedy** [comment]
Thanks @briandeconinck! I wanted to note here that when we (myself, @AngelaFowler82 and @TiffanyPender) looked at this design, our feedback lines up with yours. We've reached out to the team, and will help them with prototyping explorations around tables vs. cards for the content.

**Comment by ajakabcin** [comment]
Closing -  Must feedback was addressed in sketch file annotations for devs to ensure consistent use of H3s
